common/source/docs/common-power-module-configuration-in-mission-planner.rst

```diff
@@ -68,13 +71,18 @@ These are selected via the ``BATTx_MONITOR`` parameter for each battery monitor.
 19 	                                    :ref:`Rotoye<common-smart-battery-rotoye>`
 =================================     =============================
 
+
+.. note:: Once a specific monitor type is selected, parameters associated with that type of monitor will be revealed once parameters are refreshed. Scales and offsets, bus addresses, etc. will be displayed, as appropriate, for that monitor.
+
 Other Parameters
 ----------------
 
-- :ref:`BATT_OPTIONS<BATT_OPTIONS>` bit 0, if set, will ignore the State Of Charge field in DroneCAN moniotrs, since some do not populate this field with meaningful data.
-- :ref:`BATT_SUM_MASK<BATT_SUM_MASK>` is used if the monitor is type "10" (SumofFollowing) to select which of the higher numbered monitors values will be summed and displayed for this monitor.
+- :ref:`BATT_OPTIONS<BATT_OPTIONS>` bit 0, if set, will ignore the State Of Charge field in DroneCAN moniotrs, since some do not populate this field with meaningful data. Also various options for MPPT type monitors are provided.
```

Comment:
```suggestion
- :ref:`BATT_OPTIONS<BATT_OPTIONS>` bit 0, if set, will ignore the State Of Charge field in DroneCAN monitors, since some do not populate this field with meaningful data. Also various options for MPPT type monitors are provided.
```